### PR TITLE
Add init for n-service-worker

### PR DESCRIPTION
**Pull request by phamann**
Imports the new global service worker module and init.

*Note: This is temporarily for us to experiment and not switched on in production.*

/cc @matthew-andrews

### CODE REVIEW

**Comment by matthew-andrews** [comment]
@slightlyoff will be pleased :+1:

File: bower.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "fastclick": "^1.0.6",
     "o-hoverable": "^1.1.0",
     "o-errors": "^2.2.0",
-    "o-tracking": "^1.0.0"
+    "o-tracking": "^1.0.0",
+    "n-service-worker": "^0.0.2"
```
Comment (matthew-andrews) [inline bower.json]: 0.x.x versions are the spawn of the devil

Comment (phamann) [inline bower.json]: Can I get away with the excuse: *Experimental* :wink:

Comment (matthew-andrews) [inline bower.json]: Yeah… sure… if you hate yourself :-)
https://nodesource.com/blog/semver-tilde-and-caret#major-and-minor-zero-0-0-z-0-0-z-

screen shot 2015-08-19 at 16 59 19

**Comment by slightlyoff** [comment]
YESSSS

**Pull request merged**